Plan/api/build.gradle

```diff
@@ -5,7 +5,7 @@ dependencies {
 }
 
 compileJava {
-    options.release = 8
+    options.release = 11
```

Comment:
Can this be reverted? It was intentionally set as 8 since api is deployed as a separate artifact, so that other plugins don't need to update to 11 to use Plan API

Reply:
Ah, yes. Just saw some "release version 8 is obsolete and will be removed soon" warnings but forgot about the separate artifact